hasher-matcher-actioner/webapp/src/Upload.js

```diff
@@ -7,41 +7,57 @@ import { Link } from 'react-router-dom';
 import { Button, Col, Collapse, Image, Row } from 'react-bootstrap';
 import Spinner from 'react-bootstrap/Spinner';
 
+import { postAPI } from './Api'
+
 export default function Upload() {
-  const [ fileName, setFileName ] = useState('Select Browse to choose a file.');
-  const [ submitting, setSubmitting ] = useState(false);
-  const [ submitted, setSubmitted ] = useState(false);
+  const [fileName, setFileName] = useState('Select Browse to choose a file.');
```

Comment:
Since this is a react component, would love a more UI oriented name. eg. PreviewableImageUploadPane().